src/main/java/mate/academy/dao/jdbc/CarJdbcDao.java

```diff
@@ -0,0 +1,203 @@
+package mate.academy.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import mate.academy.dao.CarDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Car;
+import mate.academy.model.Driver;
+import mate.academy.model.Manufacturer;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class CarJdbcDao implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String insertCar = "INSERT INTO `cars` (`manufacturer_id`, `car_model`) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement insertStatement
+                        = connection.prepareStatement(insertCar, Statement.RETURN_GENERATED_KEYS)) {
+            insertStatement.setLong(1, car.getManufacturer().getId());
+            insertStatement.setString(2, car.getModel());
+            insertStatement.executeUpdate();
+            ResultSet resultSet = insertStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+            return car;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert car to the DB"
+                    + car, e);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String selectCar = "SELECT * FROM cars c "
+                + "JOIN manufacturers m ON c.manufacturer_id = m.manufacturer_id "
+                + "WHERE c.car_id = ? AND c.deleted = FALSE AND m.manufacturer_deleted = FALSE";
+        String selectDrivers = "SELECT * FROM cars_drivers cd "
+                + "JOIN drivers d ON cd.driver_id = d.driver_id "
+                + "WHERE cd.car_id = ? AND d.deleted = FALSE";
```

Comment:
let's move getting drivers from a car to a separate private method since we use it more than once